adr/0005-correlation-ids.md

```diff
@@ -0,0 +1,24 @@
+---
+creation_date: 2022-02-01
+decision_date: 2022-02-01
+status: accepted
+---
+# Use of optional Correlation IDs for tracking requests in logs
+
+## Context
+Correlation IDs (sometimes also known as "tracking IDs" or "tracing IDs") allow us to track a request across a whole interaction with our service. If there are problems we need to debug, 
+having a unique identifier per transaction that we can expose to our logging and monitoring will make it easier to track down the issue.
+
+The ID will also be included in the response header, so that consumers will also have a view of it.
```

Comment:
```suggestion
The ID will also be included in the response header, so that consumers will also have a view of it, if they had not provided one in the request.
```